File: Ludus/app/src/main/java/com/theorangeteam/ludus/android/activities/ShelfActivity.java
package com.theorangeteam.ludus.android.activities;

import android.os.Bundle;
import android.widget.TextView;

import com.theorangeteam.ludus.R;

import butterknife.BindView;
import butterknife.ButterKnife;

public class ShelfActivity extends DrawerActivity
{

    @BindView(R.id.hello)
    TextView hello;

    @Override
    public void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.content_shelf);
        ButterKnife.bind(this);
        hello.setText("Inheritance intensifies");
        this.setupDrawerLayout();
    }
}

File: Ludus/app/src/main/java/com/theorangeteam/ludus/android/activities/ShelfActivity.kt
package com.theorangeteam.ludus.android.activities

import android.os.Bundle
import android.widget.TextView

import com.theorangeteam.ludus.R

import butterknife.ButterKnife
import butterknife.bindView

class ShelfActivity : DrawerActivity() {

    val hello: TextView by bindView(R.id.hello)

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.drawer_activity_shelf)
        ButterKnife.bind(this)
        hello!!.text = "Inheritance intensifies"
        this.setupDrawerLayout()
    }
}
